modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -1552,6 +1552,10 @@ export const console: ConsoleNS = {
                         }
                     },
                     certificateSection: {
+                        heading: "Certificates",
+                        description: "The certificate information provided here is used to verify signed responses from the external Identity Provider (IdP).",
```

Comment:
Is it intentional to use the term "identity provider" given that we now use the term "connections" ?

Reply:
yes, we need to mention the exact flows since these are kind of specific cases [1] [2]

[1] - https://is.docs.wso2.com/en/latest/guides/authentication/oidc/oidc-federated-idp-initiated-logout/#configure-oidc-federated-idp-initiated-logout
[2] - https://is.docs.wso2.com/en/latest/references/app-native-authentication/#scenario-4-user-selects-federated-authentication-native-mode